File: ruby/delivery-api/GetItem.rb
# DocSection: delivery_api_get_item
# Tip: Find more about Ruby SDKs at https://docs.kontent.ai/ruby
require 'delivery-sdk-ruby'

delivery_client = Kentico::Kontent::Delivery::DeliveryClient.new project_id: '<YOUR_PROJECT_ID>'
delivery_client.item('my_article')
               .execute do |response|
                 item = response.item
               end
# EndDocSection

File: java/delivery-api/GetItem.java
// DocSection: delivery_api_get_item
// Tip: Find more about Java SDKs at https://docs.kontent.ai/java
import kentico.kontent.delivery.*;

DeliveryClient client = new DeliveryClient("<YOUR_PROJECT_ID>");

List<NameValuePair> params = DeliveryParameterBuilder.params().build();

// Create strongly typed models according to https://docs.kontent.ai/strongly-typed-models
CompletionStage<ArticleItem> item = client.getItem("my_article", ArticleItem.class, params);
// EndDocSection